css/style.css

```diff
@@ -17,3 +18,251 @@ body {
  */
 
 
+
+/* HEADER */
+
+.wrapper {
+    width: 1170px;
+    margin: 0 auto;
+}
+
+
+header{
+    height: 50px;
+    background: whitesmoke;
+    width: 100%;
+    position:relative;
+}
+
+.header-logo {
+    width: 40px;
+    height: 40px;
+    float:left;
+}
+
+nav {
+    float:right;
+    line-height: 30px;
+}
+
+nav a {
+    margin: 0 20px;
+    text-decoration: none;
+    color: rgb(129, 129, 129);
+    font-weight: 400;
+
+}
+
+nav a:nth-of-type(1) {color: black; font-weight: 500;}
+
+nav a:hover {color:chocolate;}
+
+
+
+/* MAIN (1st Section) */
+
+.banner-area {
+    position: relative;
+}
+
+
+
+
+
+.img {
+    display:flex;
+    justify-content: center;
+}
+
+.text-container {
+    display: flex;
+    flex-direction: column;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    text-align: center;
+    font-weight: 20;
+}
+
+h1 {
+    color: white;
+    font-weight: 20;
+    font-size: 65px;
+    margin-bottom: -10px;
+}
+
+p {
+    color: white;
+    font-weight: 1;
+    font-size: 35px;
+
+}
+
+
+.button {
+    display: flex;
+    align-self: center;
+    font-size: 20px;
+    color: white;
+}
+
+.button1 {
+    padding: 20px 35px;
+    background-color: chocolate;
+    border: chocolate solid;
+    border-radius: 2%;
+    margin-top: 20px;
+    font-weight: 500;
+}
+
+
+
+/* 2nd SECTION SECTION */
+
+
+.h2 {
+    display: flex;
+    position: relative;
+    justify-content: center;
+    /* text-align: center; */
+    margin-top: 100px;
+    padding: 0 100px;
+
+}
+
+
+h2 {
+    text-align: center;
+    top: 850px;
+    font-size: 55px;
+    font-weight: 20;
+    margin-top: 100px;
+}
+
+
+
+@media (max-width: 1000px){
```

Comment:
Cosmetics: It's a good practice to leave an empty line after each css rule you write:
```
h2 {
  ...
}

header {
  ...
}
```